Compilation Failure on nightly-2020-09-15

After updating from `nightly-2020-09-07` to `nightly-2020-09-15`, I get the following error when running `cargo build`:
```
2020-09-16T10:40:58.0996802Z error[E0432]: unresolved import `crate::arch::x86_64::_mm_movemask_pi8`
2020-09-16T10:40:58.0998748Z    --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86/sse.rs:47:21
2020-09-16T10:40:58.1000053Z     |
2020-09-16T10:40:58.1001068Z 38  | / macro_rules! x86_m8x8_sse_impl {
2020-09-16T10:40:58.1002117Z 39  | |     ($id:ident) => {
2020-09-16T10:40:58.1003145Z 40  | |         impl All for $id {
2020-09-16T10:40:58.1004848Z 41  | |             #[inline]
2020-09-16T10:40:58.1005878Z ...   |
2020-09-16T10:40:58.1007351Z 47  | |                 use crate::arch::x86_64::_mm_movemask_pi8;
2020-09-16T10:40:58.1008499Z     | |                     ^^^^^^^^^^^^^^^^^^^^^----------------
2020-09-16T10:40:58.1009524Z     | |                     |                    |
2020-09-16T10:40:58.1010963Z     | |                     |                    help: a similar name exists in the module: `_mm_movemask_epi8`
2020-09-16T10:40:58.1012494Z     | |                     no `_mm_movemask_pi8` in `arch::x86_64`
2020-09-16T10:40:58.1013434Z ...   |
2020-09-16T10:40:58.1014449Z 67  | |     };
2020-09-16T10:40:58.1015369Z 68  | | }
2020-09-16T10:40:58.1016401Z     | |_- in this expansion of `x86_m8x8_sse_impl!` (#3)
2020-09-16T10:40:58.1017294Z     | 
2020-09-16T10:40:58.1019971Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask.rs:41:1
2020-09-16T10:40:58.1021211Z     |
2020-09-16T10:40:58.1022043Z 41  |   impl_mask_reductions!(m8x8);
2020-09-16T10:40:58.1023022Z     |   ---------------------------- in this macro invocation (#1)
2020-09-16T10:40:58.1023993Z     | 
2020-09-16T10:40:58.1026081Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86.rs:20:1
2020-09-16T10:40:58.1029104Z     |
2020-09-16T10:40:58.1030030Z 20  | / macro_rules! x86_m8x8_impl {
2020-09-16T10:40:58.1030924Z 21  | |     ($id:ident) => {
2020-09-16T10:40:58.1031769Z 22  | |         cfg_if! {
2020-09-16T10:40:58.1032735Z 23  | |             if #[cfg(all(target_arch = "x86_64", target_feature = "sse"))] {
2020-09-16T10:40:58.1034229Z 24  | |                 x86_m8x8_sse_impl!($id);
2020-09-16T10:40:58.1035268Z     | |                 ------------------------ in this macro invocation (#3)
2020-09-16T10:40:58.1036069Z ...   |
2020-09-16T10:40:58.1036809Z 29  | |     };
2020-09-16T10:40:58.1037574Z 30  | | }
2020-09-16T10:40:58.1038556Z     | |_- in this expansion of `x86_m8x8_impl!` (#2)
2020-09-16T10:40:58.1039896Z ...
2020-09-16T10:40:58.1040769Z 141 | / macro_rules! impl_mask_reductions {
2020-09-16T10:40:58.1041711Z 142 | |     // 64-bit wide masks
2020-09-16T10:40:58.1042627Z 143 | |     (m8x8) => { x86_m8x8_impl!(m8x8); };
2020-09-16T10:40:58.1043625Z     | |                 --------------------- in this macro invocation (#2)
2020-09-16T10:40:58.1045453Z 144 | |     (m16x4) => { x86_m8x8_impl!(m16x4); };
2020-09-16T10:40:58.1046724Z ...   |
2020-09-16T10:40:58.1050136Z 193 | |     ($id:ident) => { fallback_impl!($id); };
2020-09-16T10:40:58.1051064Z 194 | | }
2020-09-16T10:40:58.1051793Z     | |_- in this expansion of `impl_mask_reductions!` (#1)
2020-09-16T10:40:58.1052148Z 
2020-09-16T10:40:58.1052991Z error[E0432]: unresolved import `crate::arch::x86_64::_mm_movemask_pi8`
2020-09-16T10:40:58.1054310Z    --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86/sse.rs:62:21
2020-09-16T10:40:58.1055218Z     |
2020-09-16T10:40:58.1055828Z 38  | / macro_rules! x86_m8x8_sse_impl {
2020-09-16T10:40:58.1056499Z 39  | |     ($id:ident) => {
2020-09-16T10:40:58.1057292Z 40  | |         impl All for $id {
2020-09-16T10:40:58.1057926Z 41  | |             #[inline]
2020-09-16T10:40:58.1058443Z ...   |
2020-09-16T10:40:58.1059359Z 62  | |                 use crate::arch::x86_64::_mm_movemask_pi8;
2020-09-16T10:40:58.1060116Z     | |                     ^^^^^^^^^^^^^^^^^^^^^----------------
2020-09-16T10:40:58.1061641Z     | |                     |                    |
2020-09-16T10:40:58.1062548Z     | |                     |                    help: a similar name exists in the module: `_mm_movemask_epi8`
2020-09-16T10:40:58.1063669Z     | |                     no `_mm_movemask_pi8` in `arch::x86_64`
2020-09-16T10:40:58.1064833Z ...   |
2020-09-16T10:40:58.1065390Z 67  | |     };
2020-09-16T10:40:58.1066006Z 68  | | }
2020-09-16T10:40:58.1066695Z     | |_- in this expansion of `x86_m8x8_sse_impl!` (#3)
2020-09-16T10:40:58.1067230Z     | 
2020-09-16T10:40:58.1068365Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask.rs:41:1
2020-09-16T10:40:58.1069218Z     |
2020-09-16T10:40:58.1069797Z 41  |   impl_mask_reductions!(m8x8);
2020-09-16T10:40:58.1070557Z     |   ---------------------------- in this macro invocation (#1)
2020-09-16T10:40:58.1071091Z     | 
2020-09-16T10:40:58.1072198Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86.rs:20:1
2020-09-16T10:40:58.1073027Z     |
2020-09-16T10:40:58.1073618Z 20  | / macro_rules! x86_m8x8_impl {
2020-09-16T10:40:58.1074280Z 21  | |     ($id:ident) => {
2020-09-16T10:40:58.1074886Z 22  | |         cfg_if! {
2020-09-16T10:40:58.1075639Z 23  | |             if #[cfg(all(target_arch = "x86_64", target_feature = "sse"))] {
2020-09-16T10:40:58.1076535Z 24  | |                 x86_m8x8_sse_impl!($id);
2020-09-16T10:40:58.1077379Z     | |                 ------------------------ in this macro invocation (#3)
2020-09-16T10:40:58.1077976Z ...   |
2020-09-16T10:40:58.1078506Z 29  | |     };
2020-09-16T10:40:58.1079198Z 30  | | }
2020-09-16T10:40:58.1079881Z     | |_- in this expansion of `x86_m8x8_impl!` (#2)
2020-09-16T10:40:58.1080389Z ...
2020-09-16T10:40:58.1083253Z 141 | / macro_rules! impl_mask_reductions {
2020-09-16T10:40:58.1084728Z 142 | |     // 64-bit wide masks
2020-09-16T10:40:58.1085477Z 143 | |     (m8x8) => { x86_m8x8_impl!(m8x8); };
2020-09-16T10:40:58.1086615Z     | |                 --------------------- in this macro invocation (#2)
2020-09-16T10:40:58.1087648Z 144 | |     (m16x4) => { x86_m8x8_impl!(m16x4); };
2020-09-16T10:40:58.1088196Z ...   |
2020-09-16T10:40:58.1088834Z 193 | |     ($id:ident) => { fallback_impl!($id); };
2020-09-16T10:40:58.1089931Z 194 | | }
2020-09-16T10:40:58.1103702Z     | |_- in this expansion of `impl_mask_reductions!` (#1)
2020-09-16T10:40:58.1104093Z 
2020-09-16T10:40:58.1106422Z error[E0432]: unresolved import `crate::arch::x86_64::_mm_movemask_pi8`
2020-09-16T10:40:58.1108085Z    --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86/sse.rs:47:21
2020-09-16T10:40:58.1108979Z     |
2020-09-16T10:40:58.1109592Z 38  | / macro_rules! x86_m8x8_sse_impl {
2020-09-16T10:40:58.1110260Z 39  | |     ($id:ident) => {
2020-09-16T10:40:58.1110907Z 40  | |         impl All for $id {
2020-09-16T10:40:58.1111547Z 41  | |             #[inline]
2020-09-16T10:40:58.1112062Z ...   |
2020-09-16T10:40:58.1113595Z 47  | |                 use crate::arch::x86_64::_mm_movemask_pi8;
2020-09-16T10:40:58.1114365Z     | |                     ^^^^^^^^^^^^^^^^^^^^^----------------
2020-09-16T10:40:58.1115025Z     | |                     |                    |
2020-09-16T10:40:58.1116064Z     | |                     |                    help: a similar name exists in the module: `_mm_movemask_epi8`
2020-09-16T10:40:58.1117230Z     | |                     no `_mm_movemask_pi8` in `arch::x86_64`
2020-09-16T10:40:58.1117800Z ...   |
2020-09-16T10:40:58.1118549Z 67  | |     };
2020-09-16T10:40:58.1119111Z 68  | | }
2020-09-16T10:40:58.1119787Z     | |_- in this expansion of `x86_m8x8_sse_impl!` (#3)
2020-09-16T10:40:58.1120318Z     | 
2020-09-16T10:40:58.1121440Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask.rs:47:1
2020-09-16T10:40:58.1122279Z     |
2020-09-16T10:40:58.1122864Z 47  |   impl_mask_reductions!(m16x4);
2020-09-16T10:40:58.1123650Z     |   ----------------------------- in this macro invocation (#1)
2020-09-16T10:40:58.1124202Z     | 
2020-09-16T10:40:58.1125304Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86.rs:20:1
2020-09-16T10:40:58.1126136Z     |
2020-09-16T10:40:58.1126725Z 20  | / macro_rules! x86_m8x8_impl {
2020-09-16T10:40:58.1127386Z 21  | |     ($id:ident) => {
2020-09-16T10:40:58.1127995Z 22  | |         cfg_if! {
2020-09-16T10:40:58.1129066Z 23  | |             if #[cfg(all(target_arch = "x86_64", target_feature = "sse"))] {
2020-09-16T10:40:58.1129884Z 24  | |                 x86_m8x8_sse_impl!($id);
2020-09-16T10:40:58.1130851Z     | |                 ------------------------ in this macro invocation (#3)
2020-09-16T10:40:58.1131463Z ...   |
2020-09-16T10:40:58.1131998Z 29  | |     };
2020-09-16T10:40:58.1132562Z 30  | | }
2020-09-16T10:40:58.1133232Z     | |_- in this expansion of `x86_m8x8_impl!` (#2)
2020-09-16T10:40:58.1133743Z ...
2020-09-16T10:40:58.1211392Z 141 | / macro_rules! impl_mask_reductions {
2020-09-16T10:40:58.1212252Z 142 | |     // 64-bit wide masks
2020-09-16T10:40:58.1214329Z 143 | |     (m8x8) => { x86_m8x8_impl!(m8x8); };
2020-09-16T10:40:58.1215042Z 144 | |     (m16x4) => { x86_m8x8_impl!(m16x4); };
2020-09-16T10:40:58.1217613Z     | |                  ---------------------- in this macro invocation (#2)
2020-09-16T10:40:58.1218310Z ...   |
2020-09-16T10:40:58.1220501Z 193 | |     ($id:ident) => { fallback_impl!($id); };
2020-09-16T10:40:58.1221382Z 194 | | }
2020-09-16T10:40:58.1222165Z     | |_- in this expansion of `impl_mask_reductions!` (#1)
2020-09-16T10:40:58.1222757Z 
2020-09-16T10:40:58.1224373Z error[E0432]: unresolved import `crate::arch::x86_64::_mm_movemask_pi8`
2020-09-16T10:40:58.1225584Z    --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86/sse.rs:62:21
2020-09-16T10:40:58.1226446Z     |
2020-09-16T10:40:58.1227056Z 38  | / macro_rules! x86_m8x8_sse_impl {
2020-09-16T10:40:58.1227726Z 39  | |     ($id:ident) => {
2020-09-16T10:40:58.1228376Z 40  | |         impl All for $id {
2020-09-16T10:40:58.1229013Z 41  | |             #[inline]
2020-09-16T10:40:58.1229534Z ...   |
2020-09-16T10:40:58.1230444Z 62  | |                 use crate::arch::x86_64::_mm_movemask_pi8;
2020-09-16T10:40:58.1231185Z     | |                     ^^^^^^^^^^^^^^^^^^^^^----------------
2020-09-16T10:40:58.1231854Z     | |                     |                    |
2020-09-16T10:40:58.1232733Z     | |                     |                    help: a similar name exists in the module: `_mm_movemask_epi8`
2020-09-16T10:40:58.1233835Z     | |                     no `_mm_movemask_pi8` in `arch::x86_64`
2020-09-16T10:40:58.1234401Z ...   |
2020-09-16T10:40:58.1234946Z 67  | |     };
2020-09-16T10:40:58.1235505Z 68  | | }
2020-09-16T10:40:58.1236803Z     | |_- in this expansion of `x86_m8x8_sse_impl!` (#3)
2020-09-16T10:40:58.1237495Z     | 
2020-09-16T10:40:58.1238642Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask.rs:47:1
2020-09-16T10:40:58.1239520Z     |
2020-09-16T10:40:58.1240112Z 47  |   impl_mask_reductions!(m16x4);
2020-09-16T10:40:58.1240882Z     |   ----------------------------- in this macro invocation (#1)
2020-09-16T10:40:58.1241442Z     | 
2020-09-16T10:40:58.1242725Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86.rs:20:1
2020-09-16T10:40:58.1243629Z     |
2020-09-16T10:40:58.1244228Z 20  | / macro_rules! x86_m8x8_impl {
2020-09-16T10:40:58.1245026Z 21  | |     ($id:ident) => {
2020-09-16T10:40:58.1245633Z 22  | |         cfg_if! {
2020-09-16T10:40:58.1246387Z 23  | |             if #[cfg(all(target_arch = "x86_64", target_feature = "sse"))] {
2020-09-16T10:40:58.1247143Z 24  | |                 x86_m8x8_sse_impl!($id);
2020-09-16T10:40:58.1247931Z     | |                 ------------------------ in this macro invocation (#3)
2020-09-16T10:40:58.1248526Z ...   |
2020-09-16T10:40:58.1249063Z 29  | |     };
2020-09-16T10:40:58.1249624Z 30  | | }
2020-09-16T10:40:58.1250697Z     | |_- in this expansion of `x86_m8x8_impl!` (#2)
2020-09-16T10:40:58.1251227Z ...
2020-09-16T10:40:58.1251861Z 141 | / macro_rules! impl_mask_reductions {
2020-09-16T10:40:58.1252582Z 142 | |     // 64-bit wide masks
2020-09-16T10:40:58.1253280Z 143 | |     (m8x8) => { x86_m8x8_impl!(m8x8); };
2020-09-16T10:40:58.1253971Z 144 | |     (m16x4) => { x86_m8x8_impl!(m16x4); };
2020-09-16T10:40:58.1254775Z     | |                  ---------------------- in this macro invocation (#2)
2020-09-16T10:40:58.1255368Z ...   |
2020-09-16T10:40:58.1256007Z 193 | |     ($id:ident) => { fallback_impl!($id); };
2020-09-16T10:40:58.1256637Z 194 | | }
2020-09-16T10:40:58.1257355Z     | |_- in this expansion of `impl_mask_reductions!` (#1)
2020-09-16T10:40:58.1257709Z 
2020-09-16T10:40:58.1258533Z error[E0432]: unresolved import `crate::arch::x86_64::_mm_movemask_pi8`
2020-09-16T10:40:58.1259695Z    --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86/sse.rs:47:21
2020-09-16T10:40:58.1260536Z     |
2020-09-16T10:40:58.1261140Z 38  | / macro_rules! x86_m8x8_sse_impl {
2020-09-16T10:40:58.1261805Z 39  | |     ($id:ident) => {
2020-09-16T10:40:58.1262572Z 40  | |         impl All for $id {
2020-09-16T10:40:58.1263257Z 41  | |             #[inline]
2020-09-16T10:40:58.1263770Z ...   |
2020-09-16T10:40:58.1264679Z 47  | |                 use crate::arch::x86_64::_mm_movemask_pi8;
2020-09-16T10:40:58.1265547Z     | |                     ^^^^^^^^^^^^^^^^^^^^^----------------
2020-09-16T10:40:58.1266205Z     | |                     |                    |
2020-09-16T10:40:58.1267064Z     | |                     |                    help: a similar name exists in the module: `_mm_movemask_epi8`
2020-09-16T10:40:58.1268180Z     | |                     no `_mm_movemask_pi8` in `arch::x86_64`
2020-09-16T10:40:58.1268741Z ...   |
2020-09-16T10:40:58.1269286Z 67  | |     };
2020-09-16T10:40:58.1269840Z 68  | | }
2020-09-16T10:40:58.1270520Z     | |_- in this expansion of `x86_m8x8_sse_impl!` (#3)
2020-09-16T10:40:58.1271051Z     | 
2020-09-16T10:40:58.1272160Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask.rs:52:1
2020-09-16T10:40:58.1272996Z     |
2020-09-16T10:40:58.1273576Z 52  |   impl_mask_reductions!(m32x2);
2020-09-16T10:40:58.1274344Z     |   ----------------------------- in this macro invocation (#1)
2020-09-16T10:40:58.1274891Z     | 
2020-09-16T10:40:58.1275994Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86.rs:20:1
2020-09-16T10:40:58.1276829Z     |
2020-09-16T10:40:58.1371143Z 20  | / macro_rules! x86_m8x8_impl {
2020-09-16T10:40:58.1371933Z 21  | |     ($id:ident) => {
2020-09-16T10:40:58.1372557Z 22  | |         cfg_if! {
2020-09-16T10:40:58.1373315Z 23  | |             if #[cfg(all(target_arch = "x86_64", target_feature = "sse"))] {
2020-09-16T10:40:58.1374080Z 24  | |                 x86_m8x8_sse_impl!($id);
2020-09-16T10:40:58.1374871Z     | |                 ------------------------ in this macro invocation (#3)
2020-09-16T10:40:58.1375475Z ...   |
2020-09-16T10:40:58.1376010Z 29  | |     };
2020-09-16T10:40:58.1377527Z 30  | | }
2020-09-16T10:40:58.1378341Z     | |_- in this expansion of `x86_m8x8_impl!` (#2)
2020-09-16T10:40:58.1379175Z ...
2020-09-16T10:40:58.1379900Z 141 | / macro_rules! impl_mask_reductions {
2020-09-16T10:40:58.1381140Z 142 | |     // 64-bit wide masks
2020-09-16T10:40:58.1381827Z 143 | |     (m8x8) => { x86_m8x8_impl!(m8x8); };
2020-09-16T10:40:58.1383078Z 144 | |     (m16x4) => { x86_m8x8_impl!(m16x4); };
2020-09-16T10:40:58.1383868Z 145 | |     (m32x2) => { x86_m8x8_impl!(m32x2); };
2020-09-16T10:40:58.1384668Z     | |                  ---------------------- in this macro invocation (#2)
2020-09-16T10:40:58.1385261Z ...   |
2020-09-16T10:40:58.1385908Z 193 | |     ($id:ident) => { fallback_impl!($id); };
2020-09-16T10:40:58.1387247Z 194 | | }
2020-09-16T10:40:58.1387972Z     | |_- in this expansion of `impl_mask_reductions!` (#1)
2020-09-16T10:40:58.1388328Z 
2020-09-16T10:40:58.1389822Z error[E0432]: unresolved import `crate::arch::x86_64::_mm_movemask_pi8`
2020-09-16T10:40:58.1391504Z    --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86/sse.rs:62:21
2020-09-16T10:40:58.1392642Z     |
2020-09-16T10:40:58.1393321Z 38  | / macro_rules! x86_m8x8_sse_impl {
2020-09-16T10:40:58.1395170Z 39  | |     ($id:ident) => {
2020-09-16T10:40:58.1395833Z 40  | |         impl All for $id {
2020-09-16T10:40:58.1396471Z 41  | |             #[inline]
2020-09-16T10:40:58.1396992Z ...   |
2020-09-16T10:40:58.1397931Z 62  | |                 use crate::arch::x86_64::_mm_movemask_pi8;
2020-09-16T10:40:58.1398678Z     | |                     ^^^^^^^^^^^^^^^^^^^^^----------------
2020-09-16T10:40:58.1399346Z     | |                     |                    |
2020-09-16T10:40:58.1400204Z     | |                     |                    help: a similar name exists in the module: `_mm_movemask_epi8`
2020-09-16T10:40:58.1401323Z     | |                     no `_mm_movemask_pi8` in `arch::x86_64`
2020-09-16T10:40:58.1402058Z ...   |
2020-09-16T10:40:58.1402657Z 67  | |     };
2020-09-16T10:40:58.1403215Z 68  | | }
2020-09-16T10:40:58.1403894Z     | |_- in this expansion of `x86_m8x8_sse_impl!` (#3)
2020-09-16T10:40:58.1404557Z     | 
2020-09-16T10:40:58.1405678Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask.rs:52:1
2020-09-16T10:40:58.1406514Z     |
2020-09-16T10:40:58.1407099Z 52  |   impl_mask_reductions!(m32x2);
2020-09-16T10:40:58.1407877Z     |   ----------------------------- in this macro invocation (#1)
2020-09-16T10:40:58.1408425Z     | 
2020-09-16T10:40:58.1409527Z    ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/codegen/reductions/mask/x86.rs:20:1
2020-09-16T10:40:58.1410525Z     |
2020-09-16T10:40:58.1411159Z 20  | / macro_rules! x86_m8x8_impl {
2020-09-16T10:40:58.1411815Z 21  | |     ($id:ident) => {
2020-09-16T10:40:58.1412439Z 22  | |         cfg_if! {
2020-09-16T10:40:58.1413196Z 23  | |             if #[cfg(all(target_arch = "x86_64", target_feature = "sse"))] {
2020-09-16T10:40:58.1413949Z 24  | |                 x86_m8x8_sse_impl!($id);
2020-09-16T10:40:58.1414733Z     | |                 ------------------------ in this macro invocation (#3)
2020-09-16T10:40:58.1415335Z ...   |
2020-09-16T10:40:58.1415864Z 29  | |     };
2020-09-16T10:40:58.1416416Z 30  | | }
2020-09-16T10:40:58.1417189Z     | |_- in this expansion of `x86_m8x8_impl!` (#2)
2020-09-16T10:40:58.1417702Z ...
2020-09-16T10:40:58.1418337Z 141 | / macro_rules! impl_mask_reductions {
2020-09-16T10:40:58.1419056Z 142 | |     // 64-bit wide masks
2020-09-16T10:40:58.1419733Z 143 | |     (m8x8) => { x86_m8x8_impl!(m8x8); };
2020-09-16T10:40:58.1420432Z 144 | |     (m16x4) => { x86_m8x8_impl!(m16x4); };
2020-09-16T10:40:58.1421122Z 145 | |     (m32x2) => { x86_m8x8_impl!(m32x2); };
2020-09-16T10:40:58.1422053Z     | |                  ---------------------- in this macro invocation (#2)
2020-09-16T10:40:58.1422691Z ...   |
2020-09-16T10:40:58.1423330Z 193 | |     ($id:ident) => { fallback_impl!($id); };
2020-09-16T10:40:58.1423954Z 194 | | }
2020-09-16T10:40:58.1424791Z     | |_- in this expansion of `impl_mask_reductions!` (#1)
```

This is on `nightly-x86_64-unknown-linux-gnu`. I'm no expert on these things, but it looks like some stuff in `core::arch` got removed - at least `_mm_movemask_pi8` is not in the nightly docs anymore.

--- a/src/codegen/reductions/mask/x86.rs
+++ b/src/codegen/reductions/mask/x86.rs
@@ -19,13 +19,7 @@ mod avx2;
 /// x86 64-bit m8x8 implementation
 macro_rules! x86_m8x8_impl {
     ($id:ident) => {
-        cfg_if! {
-            if #[cfg(all(target_arch = "x86_64", target_feature = "sse"))] {
-                x86_m8x8_sse_impl!($id);
-            } else {
-                fallback_impl!($id);
-            }
-        }
+        fallback_impl!($id);
     };
 }
 

--- a/src/codegen/reductions/mask/x86/sse.rs
+++ b/src/codegen/reductions/mask/x86/sse.rs
@@ -34,35 +34,3 @@ macro_rules! x86_m32x4_sse_impl {
         }
     };
 }
-
-macro_rules! x86_m8x8_sse_impl {
-    ($id:ident) => {
-        impl All for $id {
-            #[inline]
-            #[target_feature(enable = "sse")]
-            unsafe fn all(self) -> bool {
-                #[cfg(target_arch = "x86")]
-                use crate::arch::x86::_mm_movemask_pi8;
-                #[cfg(target_arch = "x86_64")]
-                use crate::arch::x86_64::_mm_movemask_pi8;
-                // _mm_movemask_pi8(a) creates an 8bit mask containing the most
-                // significant bit of each byte of `a`. If all bits are set,
-                // then all 8 lanes of the mask are true.
-                _mm_movemask_pi8(crate::mem::transmute(self))
-                    == u8::max_value() as i32
-            }
-        }
-        impl Any for $id {
-            #[inline]
-            #[target_feature(enable = "sse")]
-            unsafe fn any(self) -> bool {
-                #[cfg(target_arch = "x86")]
-                use crate::arch::x86::_mm_movemask_pi8;
-                #[cfg(target_arch = "x86_64")]
-                use crate::arch::x86_64::_mm_movemask_pi8;
-
-                _mm_movemask_pi8(crate::mem::transmute(self)) != 0
-            }
-        }
-    };
-}

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -209,7 +209,6 @@
     link_llvm_intrinsics,
     core_intrinsics,
     stmt_expr_attributes,
-    mmx_target_feature,
     crate_visibility_modifier,
     custom_inner_attributes
 )]

